docs/modules/ROOT/pages/integrating-with-existing-deployments-collapsed.adoc

```diff
@@ -0,0 +1,252 @@
+= Integrating CMOS with Existing Monitoring Stacks
+:icons: font
+:imagesdir: ../assets/images
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+[abstract]
+You can integrate CMOS with your existing Prometheus, Alertmanager, and Grafana monitoring system. 
+This tutorial documents how to configure each component in order to achieve this.
+
+include::partial$tutorial.adoc[]
+
+== Pre-requisites
+
+The environment you wish to integrate CMOS into must have:
+
+. At least one **Couchbase Server** node;
+. **Prometheus**, with **Alertmanager** installed;
+. **Grafana**, configured to use the Prometheus server as a Data Source;
+
+==== Optional ====
+
+- **Docker**, if using the Cluster Monitor's Docker image instead of the binary.
+
+== Installation
+
+=== Install Cluster Monitor
+
+The Cluster Monitor can be installed as a binary (`.rpm`), or by downloading a Docker image.
+
+**Binary installation**
+
+_Binary installation steps.... .rpm just install using package manager?..._
+
+**Docker installation**
+
+[NOTE]
+====
+We plan for the image to be distributed on DockerHub, but currently this is not available. 
+====
+
+Run the Docker container, using the preview image given to you by Couchbase:
+[source, console]
+----
+docker run -d --rm \
+  -p 7196:7196 \ <.>
+  --network cmos \
+  --prometheus-label-selector job=couchbase-server <.>
+  --couchbase-user Administrator
+  --couchbase-password password
+  --name cluster_monitor \
+  <YOUR IMAGE HERE>
+----
+
+ALSO ADD ALERTMANAGER URLS to configure cluster monitor to send to it
+
+<.> If you are using TLS, you will need to add `-p 7197:7197` to also expose port `7197`.
+<.> `job=couchbase-server` is a label we will create later in this tutorial when configuring Prometheus' `scrape_config` to scrape our Couchbase Server nodes.
+
+=== Configure Cluster Monitor
+
+Navigate to the Web GUI, which listens by default on:
+- HTTP: 7196
+- HTTPS: 7197
+
+Then click "Add Cluster" and enter the IP address of a node in the cluster, along with the cluster's configured username and password.
+Repeat this for every cluster you wish to add to the Cluster Monitor.
+
+=== Install the Agent on your Couchbase Server nodes
+
+[WARNING]
+====
+This agent is experimental. Please read the https://github.com/markspolakovs/yacpe/blob/master/README.md[documentation], specifically its disclaimer, before installation.
+====
+
+See https://issues.couchbase.com/browse/CMOS-210[CMOS-210], which tracks the development (including documentation) of the Agent.
+As there is currently no documentation on the planned deployment, this section cannot be written yet.
```

Comment:
Can we word this in a slightly user-friendlier style? Perhaps something like "Until this is ready, follow the instructions given to you by your Couchbase representative"?